include/fuse_lowlevel.h

```diff
@@ -1826,11 +1844,15 @@ int fuse_session_loop(struct fuse_session *se);
  * fuse_session_loop().
  *
  * @param se the session
- * @param clone_fd whether to use separate device fds for each thread
- *                 (may increase performance)
+ * @param config additional configuration parameters
```

Comment:
"additional" is a little confusing. In addition to what? Just call it "session loop configuration".